doc/api/http.md

```diff
@@ -2447,6 +2447,7 @@ changes:
     avoided. See [`--insecure-http-parser`][] for more information.
     **Default:** `false`
   * `localAddress` {string} Local interface to bind for network connections.
+  * `localPort` {number} Local port to connect from.
```

Comment:
Nit: add default value?

Reply:
It's not a constant. Is "OS selected" ok?

Reply:
If we add it, I think we should also add it here https://nodejs.org/api/net.html#net_socket_connect_options_connectlistener.